src/native/libs/System.Security.Cryptography.Native/openssl.c

```diff
@@ -668,13 +668,11 @@ BIO* CryptoNative_GetX509NameInfo(X509* x509, int32_t nameType, int32_t forIssue
                                 if (sizeof(szOidUpn) == cchLocalOid &&
                                     0 == strncmp(localOid, szOidUpn, sizeof(szOidUpn)))
                                 {
-                                    // OTHERNAME->ASN1_TYPE->union.field
-                                    if (!value->value)
+                                    if (value->value)
                                     {
-                                        return NULL;
+                                        // OTHERNAME->ASN1_TYPE->union.field
+                                        str = value->value->value.asn1_string;
```

Comment:
I assume this is because we were otherwise missing the sk_GENERAL_NAME_free at the end?

Reply:
Yep.  It turns out it's actually unreachable (value->value is never null), but if that were ever to change the code should be right.